runtime/src/accounts_db.rs

```diff
@@ -3851,10 +3852,18 @@ impl AccountsDB {
     pub fn calculate_accounts_hash_without_index(
         storages: &[SnapshotStorage],
         simple_capitalization_enabled: bool,
+        thread_pool: Option<&ThreadPool>,
     ) -> (Hash, u64) {
-        let result = Self::scan_snapshot_stores(storages, simple_capitalization_enabled);
+        let function = || {
```

Comment:
can we call it `scan_and_hash` or something more specific than `function`.

Reply:
done